src/zino/state.py

```diff
@@ -25,6 +26,9 @@
 
 config: Configuration = Configuration()
 
+# Last time the pollfile was modified
+pollfile_mtime: Optional[datetime] = None
```

Comment:
This is purely an internal value, so I'm not sure why we bother converting `mtime` from a unix timestamp to a `datetime` object at all.  We're really just verifying whether the value changed at all, so it doesn't even matter what the value represents.

Reply:
I found it easier having it as a datetime object for debugging, but if you think we should keep it as a unix timestamp, we can do that